# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/tsa/metamail/80_mm-2.7-strip.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

diff -Nur mm2.7.orig/src/Makefile mm2.7/src/Makefile
--- mm2.7.orig/src/Makefile	2003-05-15 00:11:18.000000000 +0200
+++ mm2.7/src/Makefile	2003-05-15 00:19:21.000000000 +0200
@@ -72,7 +72,7 @@
 #	install -s -c $(LOCALBINDIR) $$file
 #
 #INSTALL = cp
-INSTALL = install -c -s
+INSTALL = install -c
 
 # Root of installation tree
 INSTROOT = /usr
